Samples/WebApiSample/WebApiSample/Models/Notifications.cs

```diff
@@ -0,0 +1,21 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for
+// license information.
+
+using Microsoft.Azure.NotificationHubs;
+
+namespace AppBackend.Models
+{
+    public class Notifications
+    {
+        public static Notifications Instance = new Notifications();
+
+        public NotificationHubClient Hub { get; set; }
+
+        private Notifications()
+        {
+            Hub = NotificationHubClient.CreateClientFromConnectionString("<your hub's DefaultFullSharedAccessSignature>",
```

Comment:
Putting a comment above this would be helpful as well stating what you do in the readme that it requires full access such as Manage and Send